cf-usb/cf-usb.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+resource_types:
+- name: docker-image-resource
```

Comment:
Please add a comment about why we need this (and ideally, when we can remove it and use the built-in version).

Also, I'm pretty sure you can just call this `docker-image` and override the old version and it should still work.

Reply:
Well, I can try.

Reply:
Done with commit 345074c906484e2478bf6d1289bd535795c298ff, in testing.